docs/release-notes.md

```diff
@@ -2,30 +2,52 @@
 # Release Notes
 
 If you haven't had a chance - checkout the [Letter From Devs](./letter.md) for a message from our development team.
+
+## Announcements
+### Long Term Support Release
+The Long Term Support (LTS) release of Open AMT Cloud Toolkit is official and is planned for Q3 2021. With the release of LTS, the version of Open AMT Cloud Toolkit and all of the components will be moving to 2.0.0. Why the major version change? During the 1.X development period, we made many breaking changes as we were improving our external interfaces, database schemas, and communication protocols. Starting with 2.0, we will no longer be introducing breaking changes in minor version releases. This means that any 2.X component will be compatible with other 2.X components. With this adherence to [semantic versioning](https://semver.org/) our goal is to allow our customers the flexibility to upgrade versions with confidence and clarity of compatibility.
+
 ## Key Feature Changes for 1.4
-This section outlines key features changes between versions 1.3 and 1.4 for Open AMT Cloud Toolkit. 
+This section outlines key features changes between versions 1.3 and 1.4 for Open AMT Cloud Toolkit.
+
+### Breaking Changes
+#### MPS & RPS: 
+   - **WebSocket Authentication Updated:** Kong Authenication for Websockets is now required for KVM and SOL connections.  The same JWT that is used for REST calls is also used for KVM and SOL connections.
```

Comment:
can also might be better. Ideally its different and very short lived, i.e. like 1 min